src/standalone/Vegetation/canopy_parameterizations.jl

```diff
@@ -114,8 +125,8 @@ function plant_absorbed_ppfd(
     u₂ = b - c * a_soil
     u₃ = f + c * a_soil
 
-    s₁ = exp(-h * LAI * Ω)
-    s₂ = exp(-K * LAI * Ω)
+    s₁ = exp(-min(h * LAI * Ω, 40))
```

Comment:
where does the 40 come from?

Reply:
CLM... https://www2.cesm.ucar.edu/models/cesm2/land/CLM50_Tech_Note.pdf equation 3.40

It may not be necessary, when I had added the SAI I did so according to these equations which is why this got changed. When I took the SAI back out I probably should have put this back.